api/controllers/PersonController.coffee

```diff
@@ -9,3 +9,5 @@ module.exports =
   find: (req, res) ->
     res.json({tested: true})
 
+  fails_arrow_spacing:()->
```

Comment:
Colon assignment without proper spacing<br>Empty parameter list is forbidden<br>Function arrows (-> and =>) must be spaced properly